pages/common/josm.md

```diff
@@ -0,0 +1,32 @@
+# josm
+
+> Extensible OpenStreetMap editor for Java 8+.
+> More information: <https://josm.openstreetmap.de/>.
+
+- Launch JOSM:
+
+`josm`
+
+- Launch JOSM in maximized mode:
```

Comment:
```suggestion
- Launch JOSM in full-screen mode:
```
Does it just give a full window or is it an F11 type of full
If its not an F11, resolve this although I don't think its right using the same words between command and description.

Its like saying "Describe an egg","An egg shape"

Reply:
I don't think `maximized` is the same as `fullscreen`.

Reply:
When you run it in with --max... is it like pressing F11?

Reply:
I isn't. When using `--maximize` it is like any other regular window with panel and window decoration still visible. If you don't use it, JOSM launches even smaller, so you can drag it around on the screen.

Reply:
Then this is resolved